src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseDataPage.tsx

```diff
@@ -31,26 +32,35 @@ const ReleaseDataPage = () => {
     Promise.all([
       footnotesService.getReleaseFootnoteData(releaseId),
       permissionService.canUpdateRelease(releaseId),
-    ]).then(([{ meta, footnotes: footnotesList }, canUpdateRelease]) => {
-      setFootnotesData({
-        footnoteMeta: meta,
-        footnotes: footnotesList,
-        footnoteMetaGetters: generateFootnoteMetaMap(meta),
+      permissionService.canUpdateReleaseDataFiles(releaseId),
+    ]).then(
+      ([
+        { meta, footnotes: footnotesList },
         canUpdateRelease,
-      });
-    });
+        canUpdateReleaseFootnotes,
+      ]) => {
+        setFootnotesData({
+          footnoteMeta: meta,
+          footnotes: footnotesList,
+          footnoteMetaGetters: generateFootnoteMetaMap(meta),
+          canUpdateRelease,
+          canUpdateReleaseFootnotes,
+        });
+      },
+    );
   }, [releaseId]);
 
   useEffect(() => {
-    if (activeTab === 'footnotes') {
-      getFootnoteData();
-    }
+    getFootnoteData();
```

Comment:
Given that `getFootnoteData` is called again in Tab's `onToggle`, this could lead to a unnecessary double request.

If the tabs is already open on the Footnotes tab, then it makes sense to have it like it was before. I would probably revert this change.

Reply:
Yeah agreed but there's issues with most of the tabs in the system - often you have to refresh to update status etc. I think all that probably needs addressing on another ticket but for now this is a temp workaround & this will be superseded by the next raft of changes anyway.